### ISSUE TITLE
Go Noto Universal line spacing error

### ISSUE THREAD

**Issue by skipperon** [satbyy/go-noto-universal#41]
Hi,
I think there is an error in line spacing in Go Noto Universal - it is unnaturally big. When I compare it to other fonts and even to NotoSansRegular.ttf (https://github.com/googlefonts/noto-fonts/tree/main/hinted/ttf/NotoSans/NotoSans-Regular.ttf) so the file that is taken to build latin fonts in go-noto-universal. In the picture attached I show the comparison of Line Spacing in MS Word of NotoSansRegular and GoNotoCurrent.ttf:
![Comparison](https://user-images.githubusercontent.com/107102851/172560770-9f59dfd0-48b2-4b01-8798-6580d88ef161.JPG)
It is not only the case of MS Word, I checked that in Linux in other programs, there is also unnaturally big line spacing. 
It makes very uncomfortable to use this font and espacially in programs where I am not able to change it manually.

**Comment by thipokch** [comment]
I do agree that the line spacing is huge, but it usually happens when you merge multiple languages together.

**Comment by almarklein** [comment]
Would this problem be avoided if using the regional fonts instead of the GoNotoCurrent?

**Comment by skipperon** [comment]
Yes, but that should be working in GoNotoCurrent, otherwise there is no point in using GoNotoCurrent…

**Comment by stephen1864** [comment]
For you information, it seems Microsoft overcome this by making "UI" (user interface) fonts:

segoeui.ttf - Segoe_UI

It's total number of glyphs is similar to a normal Segoe font, but the ascender and descender are narrower.

I also found that Android (or *nix) are quite smart on line spacing. Only with special glyphs like "Ǻ" appear in a line will that line's spacing changed, otherwise other lines will keep their spacing back to A-Z level. I found it by changing a font's ascender to "A", much lower than the height of "Ǻ", and Android increase the upper height to "Ǻ" automagically when it sees this glyph, ignoring my ascender setting in the font.

**Comment by satbyy** [comment]
There is already a code snippet in the repo which tweaks line spacing. I will add the command to an "FAQ" section so that people can adjust it as per their wish.

**Comment by skipperon** [comment]
> There is already a code snippet in the repo which tweaks line spacing. I will add the command to an "FAQ" section so that people can adjust it as per their wish.

@satbyy  where is this code? I cannot find it...

**Comment by johnnyshields** [comment]
@satbyy if the script uses the values from NotoSans-Regular, then why is this issue (difference from NotoSans-Regular) occurring?
